Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs

```diff
@@ -412,20 +452,19 @@ public void Wzhzhzh(EntityUid uid, MicrowaveComponent component, EntityUid? user
                     return;
                 }
 
-                // destroy microwave
                 if (_tag.HasTag(item, "MicrowaveMachineUnsafe") || _tag.HasTag(item, "Metal"))
                 {
-                    component.Broken = true;
-                    SetAppearance(uid, MicrowaveVisualState.Broken, component);
-                    _audio.PlayPvs(component.ItemBreakSound, uid);
-                    return;
+                    component.IsSparking = true;
                 }
 
                 if (_tag.HasTag(item, "MicrowaveSelfUnsafe") || _tag.HasTag(item, "Plastic"))
                 {
                     var junk = Spawn(component.BadRecipeEntityId, Transform(uid).Coordinates);
                     _container.Insert(junk, component.Storage);
                     QueueDel(item);
+                    _sharedContainer.EmptyContainer(component.Storage);
```

Comment:
Was not returning the ingredients considered a bug? To me, it makes sense that if you microwave plastic with your ingredients, that the plastic melts all over the ingredients and wastes them.

Reply:
if you try microwaving plastic on the master branch it throws an error pretty sure.